Dockerfile.dev

```diff
@@ -7,6 +7,10 @@ ENV PYTHONUNBUFFERED 1
 # Sets the default shell to bash
 ENV SHELL /bin/bash
 
+RUN set -ex \
+    && apt update \
+    && apt-get install gcc python3-dev --yes
```

Comment:
When I tried to run tests in my environment, it complained it could not build one of the dependencies because `gcc` was not found.

Reply:
Strange, there is one other person reporting the same issue: https://github.com/django-q2/django-q2/issues/211

I haven't had this issue myself so far. But I am fine with this change.

Reply:
Maybe you have worked on this project for a long time and this was my first time, so started with everything "very" fresh:
- No Docker image (`docker rmi <image name>`, remove dangling images `docker image prune`).
- Maybe the docker builder cache keeps the old deps too (`docker builder prune`).

Just trying to figure out why you cannot reproduce it 🤔